src/controllers/duplication-controller.js

```diff
@@ -0,0 +1,64 @@
+import { models } from '../models/init-models.js'
+import { Op } from 'sequelize'
+import sequelizeConn from '../util/db.js'
+
+export async function createRequest(req, res) {
+  const projectId = req.params.projectId
+  const remarks = req.body.remarks
+  const transfer = req.body.transfer
+
+  // if user chooses to transfer(boolean) onetime media
+  const onetimeAction = transfer ? 100 : 1
+
+  try {
+    const request = models.ProjectDuplicationRequest.build({
+      project_id: projectId,
+      request_remarks: remarks,
+      status: 1,
+      user_id: req.user.user_id,
+      onetime_use_action: onetimeAction,
+      notes: remarks, // not entirely sure what values go into this and the next line
+      new_project_number: (parseInt(projectId) + 1).toString(),
```

Comment:
The `new_project_number` is not set here.

The project will need to be duplicated and the value is set by the duplication logic. You can remove this.